lua_modules/mcp23017/mcp23017.lua

```diff
@@ -17,14 +17,8 @@
 	@version 1.0.0
 ]]
 
-local i2c, string = i2c, string
-local issetBit = bit.isset
-local setBit = bit.set
-local clearBit = bit.clear
-
--- metatable
-local mcp23017 = {}
-mcp23017.__index = mcp23017
+local i2c, string, issetBit, setBit, clearBit, error = i2c, string, bit.isset, bit.set, bit.clear, error
+local isINPUT, isGPB, isHIGH = true, true, true
```

Comment:
These constants are probably just going to make things slower, sadly.  While I appreciate that named enumerations are better than booleans in many cases, it's probably best to just name the arguments suggestively.

Reply:
Isn't premature optimization the [root of all evil](https://ubiquity.acm.org/article.cfm?id=1513451)? I personally value more legible code - that in itself is subjective - higher unless I can prove the lines in question are "too slow" (again, often subjective).

Reply:
Maybe so, but `function(foo) if foo == isINPUT then ... end end` is less clear to me than `function(isINPUT) if isINPUT then ... end`.